packages/python/accern_xyme/accern_xyme.py

```diff
@@ -2110,6 +2113,17 @@ def get_path(self, *path: str) -> 'BlobHandle':
         return BlobHandle(
             self._client, f"{self._uri}/{'/'.join(path)}", is_full=True)
 
+    def get_parent(self) -> 'BlobHandle':
+        if self._parent is None:
+            parsed = parse.urlparse(self._uri)
+            path = "/".join(parsed.path.split("/")[1:3])
```

Comment:
how can you do `1:3`? what is 0?

Reply:
the 0 is the "/" 
when I do the `geturl()` they add the "/"
but I'll return the 0 just to make sure.

Reply:
```
>>> "/".join(pathlib.PurePath(parsed.path).parts[1:3])
```